src/modules/tokens/views/Accounts/Accounts.tsx

```diff
@@ -99,12 +112,40 @@ export const Accounts: () => JSX.Element = () => {
     }
   }, [lastEvent]);
 
+  useEffect(() => {
+    // set filters if they are present in the URL
+    if (filterQuery.length !== 0) {
+      setActiveFilters(filterQuery as string[]);
+    }
+  }, [setActiveFilters, filterQuery]);
+
+  useEffect(() => {
+    //set query param state
+    setFilterQuery(activeFilters, 'replaceIn');
+    if (activeFilters.length === 0) {
+      setFilterString('');
+      return;
+    }
+
+    setFilterString(`&${activeFilters.join('&')}`);
+  }, [activeFilters, setFilterQuery]);
+
+  const handleOpenFilter = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setFilterAnchor(event.currentTarget);
+  };
+
+  const handleAddFilter = (filter: string) => {
+    setActiveFilters((activeFilters) => [...activeFilters, filter]);
+  };
+
+  const filterFields = ['key', 'namespace', 'updated'];
```

Comment:
Isn't the entire UI page filtered by namespace at a high level? Probably doesn't make sense to have it in any of the table filters if so.